src/foam/u2/crunch/CrunchController.js

```diff
@@ -355,12 +357,14 @@ foam.CLASS({
       var p = Promise.resolve(true);
 
       intercept.capables.forEach(capable => {
-        var seq = this.createCapableWizardSequence(intercept, capable);
-        p = p.then(() => {
-          return seq.execute().then(x => {
-            return x;
+        capable.capabilityIds.forEach((c) => {
+          var seq = this.createCapableWizardSequence(intercept, capable, c);
+          p = p.then(() => {
+            return seq.execute().then(x => {
+              return x;
+            });
```

Comment:
should be the same as the following?
```suggestion
            return seq.execute().then(x => x);
```

Reply:
alternately this method could be rewritten with async/await (keeping in mind that `forEach` then can't be used)